medallion/test/data/default_data.json

```diff
@@ -202,6 +202,19 @@
                         "name": "TLP:GREEN",
                         "spec_version": "2.1",
                         "type": "marking-definition"
+                    },
+			{
+                        "created": "2017-01-27T13:49:53.997Z",
+                        "description": "Poison Ivy",
+                        "id": "malware--c0931cc6-c75e-47e5-9036-78fabc95d4ec",
+                        "is_family": true,
+                        "malware_types": [
+                            "remote-access-trojan"
+                        ],
+                        "modified": "2018-02-23T18:30:00.000Z",
+                        "name": "Poison Ivy",
+                        "spec_version": "2.0",
```

Comment:
This property can be removed completely. This will better represent a 2.0 object.